transmit-lib/src/cmr/transmit/kms.clj

```diff
@@ -34,19 +34,21 @@
    :spatial-keywords :uuid
    :science-keywords :uuid
    :concepts :short-name
-   :iso-topic-categories :short-name})
+   :iso-topic-categories :uuid
+   :related-urls :uuid})
 
 (def keyword-scheme->gcmd-resource-name
   "Maps each keyword scheme to the GCMD resource name"
-  {:providers "providers/providers.csv"
-   :platforms "platforms/platforms.csv"
-   :instruments "instruments/instruments.csv"
-   :projects "projects/projects.csv"
-   :temporal-keywords "temporalresolutionrange/temporalresolutionrange.csv"
-   :spatial-keywords "locations/locations.csv"
-   :science-keywords "sciencekeywords/sciencekeywords.csv"
-   :concepts "concepts/concepts.xml"
-   :iso-topic-categories "concepts/concepts.xml"})
+  {:providers "providers?format=csv"
```

Comment:
Since they all require the format query parameter you could remove that part from this map and just add the query parameter `?format=csv` when making the call.

Reply:
I will be in here again when I really add the related-url keywords and can update it then.